airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json

```
@@ -8,6 +8,8 @@
   },
   "requiredAction_one": "待响应的任务实例",
```

```suggestion
  "requiredAction_one": "待操作的任务实例",
```

`requiredAction` -> `待操作的`
and all others below, you get the idea